catalogo-arqueologico/src/pages/Home/Home.jsx

```diff
@@ -1,12 +1,65 @@
 import React from 'react';
+import Typography from "@mui/material/Typography";
+import { styled } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import { Stack } from "@mui/material";
+import { Link } from "react-router-dom";
 
 const Home = () => {
     return (
-        <div>
-            <h1>Welcome to the Home page!</h1>
-            {/* Add your content here */}
-        </div>
+        <CustomStack>
+            <CustomBox>
```

Comment:
I recommend you to use a `<Container>` instead. It applies horizontal padding by default